src/components/Comment.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from 'react';
+import { Avatar } from '@mui/material';
+
+
+interface CommentProps {
+    avatar: string, 
+    username: string,
+    text: string,
+    creationDate: string
+}
+
+export function Comment(props: any) {
```

Comment:
Please use CommentProps interface instead of any to properly type props, (you can declare component as arrow function and type it as React.FC<CommentProps>)